### DISCUSSION TITLE
Disabling dashboard without using API key?

### DISCUSSION THREAD

**Discussion by timheuer** [qdrant/qdrant#3901] [General]
Is there a way to disable the dashboard completely (not just secure using API key)?

**Comment by timvisee** [comment]
Though we don't have a simple toggle to disable it, there is a way to do it.

You can configure `service.static_content_dir` to some path that does not exist. It'll try to serve the dashboard files from there. If there's nothing there it'll disable the dashboard. You can set this in the `config.yaml`.

Alternatively you can set `QDRANT__SERVICE__STATIC_CONTENT_DIR=/nothing` as environment variable to achieve the same.

If you're running a binary that you've compiled yourself, you can simply remove any `static` directory you might have in that same directory.

**Reply by timheuer** [reply]
Thank you, on Discord it was suggested `QDRANT__SERVICE__ENABLE_STATIC_CONTENT=0` which also worked for me.

**Reply by timvisee** [reply]
That option is better.

I wasn't aware we had it available. Happy to see you got it resolved. :+1:
